_data/banking.yml

```
@@ -208,26 +263,34 @@ websites:
       - proprietary
       - hardware
     doc: https://www.bbbank.de/produkte/konten-und-karten/online-banking/tan-verfahren.html
+    regions:
+      - de
 
   - name: BBVA Argentina
     url: https://www.bbva.com.ar/
     img: bbva.png
     facebook: bbva.argentina
     twitter: bbva_argentina
     lang: es
+    regions:
+      - ar
 
   - name: BBVA Compass
     url: https://www.bbvacompass.com/
     img: bbva.png
     facebook: bbvacompass
     twitter: BBVACompass
+    regions:
+      - ar
```

```suggestion
  - name: BBVA US
    url: https://www.bbvausa.com/
    img: bbva.png
    facebook: bbvaintheusa
    twitter: BBVA_USA
    regions:
      - us
```
BBVA Compass is the US branch of BBVA Group. They've rebranded to BBVA (without the Compass name) worldwide, so we should update this entry to reflect that.

Fun fact: BBVA used to have a [different brand name](https://www.underconsideration.com/brandnew/archives/new_logo_for_bbva.php) for each country they had branches in, including BBVA Compass in the U.S., BBVA Bancomer in Mexico, BBVA Francés in Argentina, and BBVA Continental in Peru. After they did an [identity redesign](https://www.bbva.com/en/2019-rebrand-designing-the-new-identity/), they decided to [deploy a unified brand everywhere](https://www.bbva.com/en/bbva-deploys-its-new-brand-identity-worldwide/) and get rid of the separate brand names.